src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -100,8 +103,60 @@ public static SourceRepository GetLocalRepository()
         public static IEnumerable<SourceRepository> GetRemoteRepositories(ChocolateyConfiguration configuration, ILogger nugetLogger, IFileSystem filesystem)
 #pragma warning restore IDE0060 // unused method parameter (nugetLogger)
         {
+            // As this is a static method, we need to call the global SimpleInjector container to get a registered service.
+            var collectorService = SimpleInjectorContainer.Container.GetInstance<IProcessCollectorService>();
+            var processTree = collectorService.GetProcessesTree();

```

Comment:
I think this maybe should not completely error out if this fails, we should catch the exception and handle it here. This is good metadata to have, but it's not critical to the core functionality of things and shouldn't prevent the general operation of choco if it fails.